gulpfile.js

```diff
@@ -247,6 +250,23 @@ gulp.task("link-bugs", function (done) {
     exec(bin("linkify-bugs.sh"), [path.join(SOURCE_DIR, "_posts")], done);
 });
 
+gulp.task('lint', function() {
+    return gulp.src(path.join(LINT_DIR, "**", "*.html"))
+        .pipe(htmllint());
+});
+
+gulp.task('checklinks', function(cb) {
+  crawler.crawl('http://localhost:3000/')
+    .on('fetch404', function(queueItem, response) {
```

Comment:
Also, in our current website - almost all the broken links point to `#` which is technically valid but redundant- wonder if this is really caught by this tool - since it's not really a 404.